<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ECT系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/漂亮ico/媒体/c6.ico</normaloff>:/漂亮ico/媒体/c6.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_save_location"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="action_capcolu"/>
    <addaction name="action_conline"/>
    <addaction name="action_concir"/>
    <addaction name="action_conrect"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>实时测量</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="tearOffEnabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>保存数据类别</string>
     </property>
     <property name="separatorsCollapsible">
      <bool>false</bool>
     </property>
     <property name="toolTipsVisible">
      <bool>false</bool>
     </property>
     <addaction name="action_fltcapdata_dat"/>
     <addaction name="action_fltcapdata_txt"/>
     <addaction name="action_intcapdata_txt"/>
     <addaction name="action_fltcondata_txt"/>
     <addaction name="action_fltconline_txt"/>
     <addaction name="action_fltconcir_txt"/>
    </widget>
    <addaction name="action_set_commun"/>
    <addaction name="separator"/>
    <addaction name="action_start_commun"/>
    <addaction name="action_stop_commun"/>
    <addaction name="separator"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>历史数据</string>
    </property>
    <addaction name="action_open_file"/>
    <addaction name="separator"/>
    <addaction name="action_startfile"/>
    <addaction name="action_pausefile"/>
    <addaction name="action_stopfile"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="whatsThis">
    <string extracomment="选择要打开的数据文件。"/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_start_commun"/>
   <addaction name="action_stop_commun"/>
   <addaction name="separator"/>
   <addaction name="action_capcolu"/>
   <addaction name="action_conline"/>
   <addaction name="action_concir"/>
   <addaction name="action_conrect"/>
   <addaction name="separator"/>
   <addaction name="action_open_file"/>
   <addaction name="action_startfile"/>
   <addaction name="action_pausefile"/>
   <addaction name="action_stopfile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_staus">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>140</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>状态</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QTextEdit" name="textEdit_timeshow">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>40</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;time:00:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_filepath">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>70</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;file:/data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_player">
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>140</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>回放速度</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <widget class="QDial" name="dial_player">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>20</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchTarget">
      <double>2.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>0</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_open_file">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/文件操作/folder.png</normaloff>:/漂亮ico/文件操作/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_capcolu">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>电容数据</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_conline">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>浓度曲线</string>
   </property>
  </action>
  <action name="action_concir">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>圆截面浓度</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_set_commun">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/文件操作/gear.png</normaloff>:/漂亮ico/文件操作/gear.png</iconset>
   </property>
   <property name="text">
    <string>连接下位机</string>
   </property>
  </action>
  <action name="action_save_location">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/文件操作/save_as.ico</normaloff>:/漂亮ico/文件操作/save_as.ico</iconset>
   </property>
   <property name="text">
    <string>保存位置</string>
   </property>
  </action>
  <action name="action_start_commun">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/文件操作/media_playback_start.ico</normaloff>:/漂亮ico/文件操作/media_playback_start.ico</iconset>
   </property>
   <property name="text">
    <string>开始测量</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_stop_commun">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/文件操作/delete_2.png</normaloff>:/漂亮ico/文件操作/delete_2.png</iconset>
   </property>
   <property name="text">
    <string>停止测量</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_startfile">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/媒体/play_24.png</normaloff>:/漂亮ico/媒体/play_24.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="whatsThis">
    <string extracomment="开始读取历史数据。"/>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="action_pausefile">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/媒体/pause_24.png</normaloff>:/漂亮ico/媒体/pause_24.png</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="whatsThis">
    <string extracomment="暂停读取，按开始按钮继续。"/>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_stopfile">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/漂亮ico/媒体/stop.png</normaloff>:/漂亮ico/媒体/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="whatsThis">
    <string extracomment="停止读取，不可继续。"/>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="action_conrect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>矩截面浓度</string>
   </property>
  </action>
  <action name="actionSs">
   <property name="text">
    <string>浓度曲线</string>
   </property>
  </action>
  <action name="actionTj">
   <property name="text">
    <string>浓度分布</string>
   </property>
  </action>
  <action name="action_read_cali">
   <property name="text">
    <string>读取标定</string>
   </property>
  </action>
  <action name="action_fltcapdata_dat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>电容数据(dat)</string>
   </property>
  </action>
  <action name="action_fltcapdata_txt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>电容数据(txt)</string>
   </property>
  </action>
  <action name="action_fltcondata_txt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>浓度数据</string>
   </property>
  </action>
  <action name="action_fltconline_txt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>圆截面浓度</string>
   </property>
  </action>
  <action name="action_fltconcir_txt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>矩截面浓度</string>
   </property>
  </action>
  <action name="action_intcapdata_txt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>整数数据(txt)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textEdit_timeshow</tabstop>
  <tabstop>textEdit_filepath</tabstop>
  <tabstop>dial_player</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
